tor_core/helpers.py

```diff
@@ -1,12 +1,25 @@
 import logging
 import re
 import sys
+import time
 
 import prawcore
 import requests
 
 from tor_core import __version__
 from tor_core.strings import bot_footer
+from tor_core.config import config
+
+subreddit_regex = re.compile(
+    'reddit.com\/r\/([a-z0-9\-\_\+]+)',
+    flags=re.IGNORECASE
+)
+
+default_exceptions = (
```

Comment:
I'm curious why this might be `default_exceptions` instead of `praw_exceptions`. I do like how it's used below though.